.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     # Checks-out the repo. More at: https://github.com/actions/checkout
     - uses: actions/checkout@v2
     - name: Select Xcode version
-      run: sudo xcode-select -switch /Applications/Xcode_11.4.app
+      run: sudo xcode-select -switch /Applications/Xcode_12.app
```

Comment:
are we considering this a breaking change in our release versioning?

Reply:
@dfed we are but we're still pre-production https://github.com/fdiaz/SwiftInspector/tags

Reply:
😅 yes I know but this should likely be a feature bump given that we're pre-prod